packages/appkit-new/src/client.ts

```diff
@@ -1580,7 +1580,8 @@ export class AppKit {
       }
     }
 
-    this.universalProvider = await UniversalProvider.init(universalProviderOptions)
+    this.universalProvider =
```

Comment:
I think we could avoid making changes to `*-new` packages until we start working on Reskin to keep the PRs smaller